src/main/java/teammates/logic/api/Logic.java

```diff
@@ -1231,6 +1231,24 @@ public void sendReminderForFeedbackSessionParticularUsers(String courseId,
                                                                              feedbackSessionName,
                                                                              usersToRemind);
     }
+    
+    /**
+     * Sends confirmation email for submission to user. <br>
+     * Preconditions: <br>
+     * * All parameters are non-null. <br>
+     */
+    public void sendConfirmationEmailForSubmission(String courseId, String feedbackSessionName,
+                                                   String userToSendFeedbackSubmissionConfirmationEmail)
+            throws EntityDoesNotExistException {
+ 
+        Assumption.assertNotNull(ERROR_NULL_PARAMETER, courseId);
+        Assumption.assertNotNull(ERROR_NULL_PARAMETER, feedbackSessionName);
+        Assumption.assertNotNull(ERROR_NULL_PARAMETER, userToSendFeedbackSubmissionConfirmationEmail);
+
```

Comment:
@wkurniawan07 Can you help me with this ? I am getting failed tests due to this.
We are not passing "user" parameter from tests. So, I tried to add  `Const.ParamsNames.USER_ID, fr.giver` in submissionParams. but that causes authentication failure. As for authentication googleId of user is used, and in `FeedbackResponseAttribute` email id is used to track the giver.